src/index.js

```diff
@@ -25,6 +25,7 @@ export const DEFAULT_CONFIG = {
   launchUrls: [],
   personalization: true,
   personalizationTimeout: 1000,
+  performanceOptimized: true,
```

Comment:
Setting new parameter to toggle between the manual events and the atomic `renderDecisions:true` approach